plugins/reachability/src/main/java/org/qbicc/plugin/reachability/ReachabilityInfo.java

```diff
@@ -107,10 +107,6 @@ public static void forceCoreClassesReachableBuildTimeInit(CompilationContext ctx
         forceCoreClassesReachable(ctxt, true);
     }
 
-    public static void forceCoreClassesReachableRunTimeInit(CompilationContext ctxt) {
-        forceCoreClassesReachable(ctxt, false);
-    }
-
     private static void forceCoreClassesReachable(CompilationContext ctxt, boolean buildTimeInit) {
```

Comment:
We could "constant-propagate" `buildTimeInit` is `true` into the ReachabilityBasicBlock builder and enable some minor cleanups

Reply:
I tried to minimize the cleanups because this PR had a potential to live for a long time, and it would get annoying to rebase if other refactors would happen in the meantime. I'm in favor of following up with cleanups though.